products/bbc.toml

```
@@ -0,0 +1,75 @@
+name = "BBC"
+description = "The British Broadcasting Corporation is a British public service broadcaster."
+slug = "bbc"
+hostnames = ["bbc.co.uk", "bbc.com"]
+sources = ["https://www.bbc.co.uk/usingthebbc/privacy-policy"]
+contributors = ["Wingo5315"]
+
+[rubric.behavioral-marketing]
+value = "yes-opt-in"
+citations = ["We\'ll only send you marketing emails or contact you about BBC programmes, services and your views on issues about the BBC if you\'ve agreed to this."]
+
+[rubric.data-breaches]
+value = "no"
+citations = [""]
+
+[rubric.data-collection-reasoning]
+value = "mostly"
+citations = ["We have to have a valid reason to use your personal information. It's called the \'lawful basis for processing\'. Sometimes we might ask your permission to do things, like when you subscribe to an email. Other times, when you\'d reasonably expect us to use your personal information, we don't ask your permission, but only when the law says it\'s fine to use it, and it fits with the rights you have."]
+notes = ["The full list of why the BBC collects personal information is in Section 6 of the Privacy Policy."]
+
+[rubric.data-deletion]
+value = "yes-automated"
+citations = ["This depends on what information you're talking about. You can delete your account. Your account information is immediately deleted. Keep in mind we keep a record of how you've used our services, but this information can't be linked back to you (and) we also keep anything you\'ve uploaded or commented on. We might have collected other personal information that you might ask to be deleted that has nothing to do with BBC account."]
+
+[rubric.history]
+value = "last-modified"
+citations = [""]
+
+[rubric.law-enforcement]
+value = "not-specified"
+citations = ["Sometimes by law we have to pass on your information to other organisations. We might also share your information if we have to by law, or when we need to protect you or other people from harm."]
+
+[rubric.list-collected]
+value = "generally"
+citations = ["We\'ll give you details about why we need your personal information and how we'll use it before you begin, unless it\'s obvious.
```

If you want to include multi-line string, you have to include triple-quotes (see my suggestion).
```suggestion
citations = ["""We\'ll give you details about why we need your personal information and how we'll use it before you begin, unless it\'s obvious.
```